app/build.gradle.kts

```diff
@@ -0,0 +1,77 @@
+plugins {
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.kotlin.android)
+}
+
+android {
+    namespace = "com.scottyab.rootbeer.sample"
+    compileSdk = libs.versions.android.compile.sdk.get().toInt()
+    buildToolsVersion = libs.versions.android.build.tools.get()
+
+    defaultConfig {
+        applicationId = "com.scottyab.rootbeer.sample"
+        minSdk = libs.versions.android.min.sdk.get().toInt()
+        targetSdk = libs.versions.android.target.sdk.get().toInt()
+        versionName = "0.9"
```

Comment:
We're just missing `versionCode = 11` otherwise the generated .apk is `RootBeerSample-0.9-[null]-release.apk`

Reply:
Fixed both versionName and versionCode, now they are specified using properties. As far as I understood it was the initial intention, please correct me if I'm wrong.